### PR TITLE
CASMCVT-250 : Update documentation for latest CDS changes

### PR DESCRIPTION
**Pull request by abhishek-g-suresh**
## Summary and Scope

This PR is a update to the CDS feature for CSM 1.5. It updates some bug fixes and so on and provides some corrective/troubleshooting instructions.

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_
              backwards incompatible

## Issues and Related PRs

* Resolves [CASMCVT-250](https://jira-pro.it.hpe.com:8443/browse/CASMCVT-250)

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: install/Create_System_Configuration_Using_Cluster_Discovery_Service.md [hunk 1]
```diff
@@ -180,15 +180,22 @@ The following steps verify the status and lists the IP addresses of nodes, fabri
...
 
    ```bash
-   "${CVT_PATH}"/cds/discover_status.py nodes --out
+   "${CVT_PATH}"/cds/discover_status.py nodes -o
    ```
 
+   > **NOTE**:
+   >
+   > - Ensure that all the NCNs and CNs are discovered and listed. If any nodes are not discovered, identify the nodes using the MAC address and verify the switch connectivity of those nodes.
+   > - If the discover status step is rerun after a long duration, make sure that to start the procedure from the beginning of [Configure the management network](#2-configure-the-management-network) section.
```
Comment (sfkramer) [inline install/Create_System_Configuration_Using_Cluster_Discovery_Service.md]: Update:
If the discover status step is rerun after a long duration, make sure that to start the procedure from the beginning of [Configure the management network](#2-configure-the-management-network) section.

To:
If the discover status step is rerun after a long duration, start the procedure from the beginning of section [Configure the management network](#2-configure-the-management-network).

**Review by sfkramer — COMMENTED** [review]
I left one comment for a needed update.

**Comment by abhishek-g-suresh** [comment]
> I left one comment for a needed update.

Addressed the comment.

**Review by shivaprasad-metimath — APPROVED** [review]
looks good

**Review by annapoorna-s-alt — APPROVED** [review]
Looks good to me.

**Pull request merged**